pkg/chartutil/expand.go

```diff
@@ -45,9 +45,6 @@ func Expand(dir string, r io.Reader) error {
 			if err := yaml.Unmarshal(file.Data, ch); err != nil {
 				return errors.Wrap(err, "cannot load Chart.yaml")
 			}
-			if err != nil {
```

Comment:
would you mind reverting this? Seems like this isn't relevant to the change at hand.

Reply:
Hmm looks like it was a lint error, I must have a different version locally. I'll revert